libfreerdp/core/nla.c

```diff
@@ -45,6 +46,11 @@
 
 #include "nla.h"
 
+#include <pkcs11-helper-1.0/pkcs11.h>
```

Comment:
include necessary to compile even without smartcard logon. But removed from here because included previously in smartcardlogon.h.